### DISCUSSION TITLE
set lower/upper bound dynamically

### DISCUSSION THREAD

**Discussion by schneich** [kalkih/mini-graph-card#970] [Ideas]
Hi,

I did a couple of graphs with the mini-graph-card. Very handy, thank you very much.

Currently, I am comparing indoor temperature to outdoor temperature. Because outside temperature is heavily fluctuating, I would like to set the upper/lower bound dynamically -> based on a template. Is it possible? (my first tests did not work...)

for example: `{{ states('sensor.dachzimmer_temperatur') | float + 5 }}`

![grafik](https://github.com/kalkih/mini-graph-card/assets/84641760/b1f416a5-a754-404f-90fd-8411d11c342d)

**Comment by ildar170975** [comment]
> for example: `{{ states('sensor.dachzimmer_temperatur') | float + 5 }}`

Templates are NOT supported in the card.
Suggest you either to start monitoring https://github.com/kalkih/mini-graph-card/issues/697 or contribute corresponding additions to software.

**Comment by schneich** [comment]
Thank you for pointing me to that feature request. I added myself there.
